Atomic/syscontainers.py

```diff
@@ -1550,6 +1562,32 @@ def prune_ostree_images(self):
         refs = {}
         app_refs = []
 
+        # Check for dangling checkouts that couldn't be deleted on uninstall.
+        checkouts = self._get_system_checkout_path()
+        for i in os.listdir(checkouts):
+            if i[0] == '.':
+                continue
+            if i[-2:] not in [".0", ".1"]:
+                continue
+
+            container = i[:-2]
+
+            # Check if the container is still installed.
+            if os.path.lexists(os.path.join(checkouts, container)):
+                continue
+
+            fullpath = os.path.join(checkouts, i)
+
```

Comment:
Small nit, too much extra space here and there

Reply:
Personally I like the space before and after the line as it breaks up the two checking blocks. However, I think it could be pulled down to be just above the second check block as it's related.

Reply:
thanks, I've amended the last fixup patch and have now:

```
            fullpath = os.path.join(checkouts, i)
            if not os.path.isdir(fullpath):
                continue
```

Indeed it looks better!  I usually don't leave a space between a declaration and `if ...` using the variable itself